### PR TITLE
Add fatality notice format

### PR DESCRIPTION
**Pull request by nickcolley**
Depends on https://github.com/alphagov/govuk-content-schemas/pull/316 being merged for tests.

Screenshot of example content:

fatality-notice-format-render-example

Paired with @fofr 🙉 

Live example:
https://www.gov.uk/government/fatalities/flight-lieutenants-alan-scott-and-geraint-roberts-have-died

Associated ticket: https://trello.com/c/0UAMj5gj/407-8-fatality-notice-migration-implement-publishing-of-format-to-publishing-api

### CODE REVIEW

File: app/presenters/fatality_notice_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class FatalityNoticePresenter < ContentItemPresenter
+  include ActionView::Helpers::UrlHelper
```
Comment (boffbowsh) [inline app/presenters/fatality_notice_presenter.rb]: Is this needed?

Comment (boffbowsh) [inline app/presenters/fatality_notice_presenter.rb]: `extend` this in `Linkable` instead

**Comment by boffbowsh** [comment]
Tiny comment but it can be addressed with the other presenters in a different PR.

File: test/test_helper.rb [hunk 2]
```diff
@@ -44,9 +44,10 @@ def assert_component_parameter(component, label, value)
     end
   end
 
-  def assert_has_component_title(title)
+  def assert_has_component_title(title, context = nil)
```
Comment (dsingleton) [inline test/test_helper.rb]: 👍

**Pull request merged**